packages/server/src/schema.ts

```diff
@@ -151,7 +151,24 @@ export const schema = makeSchema({
       }
     }),
     objectType({
-      name: "Test",
+      name: "bundler",
```

Comment:
This should be Pascal cased since it is effectively a type name and will correspond to a type declaration in our graphql. E.g.

```graphql
type bundler {
}
```

Reply:
Oh yeah, typo from me.